tests/main/interfaces-audio-playback-record/task.yaml

```diff
@@ -0,0 +1,112 @@
+summary: Ensure that the audio-playback/record interface works
+
+# Only classic Ubuntu has the pulseaudio mediation patches
+systems: [ ubuntu-1*-*64 ]
+
+environment:
+    PLAY_FILE: "/snap/test-snapd-audio-record/current/usr/share/sounds/alsa/Noise.wav"
+    # today, only 19.04 and 19.10 have a mediating pulseaudio
+    EXFAIL: "ubuntu-14 ubuntu-16 ubuntu-18"
+
+prepare: |
+    #shellcheck source=tests/lib/pkgdb.sh
+    . "$TESTSLIB"/pkgdb.sh
+    snap install --edge test-snapd-audio-record
+
+    apt-get update
+    apt-get install -y pulseaudio pulseaudio-utils
+
+    echo "Create XDG_RUNTIME_DIR=/run/user/12345"
+    # shellcheck disable=SC2174
+    mkdir -m 700 -p /run/user/12345 || true
+    chown test:test /run/user/12345
+
+restore: |
+    HOME=/home/test XDG_RUNTIME_DIR=/run/user/12345 su -p -c "pulseaudio --kill" test || true
+    snap remove test-snapd-audio-record
+    apt-get autoremove --purge -y pulseaudio pulseaudio-utils
```

Comment:
Do we need `pulseaudio-utils`?

Reply:
Yes, for paplay in "The unconfined user can play audio"